src/types/config.ts

```diff
@@ -20,15 +20,14 @@ export interface GnosisSafeConfig {
 }
 
 export interface zNAConfig {
-  // address to zDAOCore contract
-  contract: string;
-  // web3 provider
-  provider: ethers.providers.Provider;
+  // subgraphUri where indexed zDAORegistry
+  subgraphUri: string;
```

Comment:
Can we include in documentation or in a comment default values for this?

Reply:
This value is set by sdk in `config/index.ts`, I will add comment too